cmd/snap-preseed/preseed_linux.go

```diff
@@ -262,6 +450,54 @@ func prepareChroot(preseedChroot string) (*targetSnapdInfo, func(), error) {
 	}, nil
 }
 
+type preseedFilePatterns struct {
+	Exclude []string `json:"exclude"`
+	Include []string `json:"include"`
+}
+
+func createPreseedArtifact(opts *PreseedOpts) error {
+	artifactPath := filepath.Join(opts.PrepareImageDir, "system-seed", "systems", opts.SystemLabel, "preseed.tgz")
+	systemData := filepath.Join(opts.WritableDir, "system-data")
+
+	patternsFile := filepath.Join(systemData, "var/lib/snapd/preseed-export.json")
```

Comment:
it's written by snapd when it ends preseeding (respective PR is not proposed yet).  It's based on static definition in the snapd code; it could be removed after snap-preseed finished.

Reply:
Correct, it doesn't go into the artifact (because it's not on the list of files to include from `/var/lib/snapd/`).